src/SeqViz/Circular/AnnotationsViewer.tsx

```diff
@@ -2,10 +2,38 @@ import * as React from "react";
 
 import CentralIndexContext from "../handlers/centralIndex";
 import { COLOR_BORDER_MAP, darkerColor } from "../../utils/colors";
+import { Coor, SizeType } from "./Circular";
+import { inputRefFuncType } from "../Linear/SeqBlock/Translations";
+import { Annotation, Label } from "../../part";
+
+interface AnnotationsViewerProps {
+  radius: number;
+  center: Coor;
+  lineHeight: number;
+  seqLength: number;
+  findCoor: (index: number, radius: number, rotate?: boolean) => Coor;
+  getRotation: (index: number) => string;
+  generateArc: (args: {
+    innerRadius: number;
+    outerRadius: number;
+    length: number;
+    largeArc: boolean; // see svg.arc large-arc-flag
+    sweepFWD?: boolean;
+    arrowFWD?: boolean;
+    arrowREV?: boolean;
+    offset?: number;
+  }) => string;
+  rotateCoor: (coor: Coor, degrees: number) => Coor;
+  inputRef: inputRefFuncType;
+  annotations: Annotation[];
+  size: SizeType;
+  rowsToSkip: number;
+  inlinedAnnotations: Label[];
+}
 
 /**
  * Used to build up all the path elements. Does not include a display
- * of the annotation name or a line connecting name to annotation
+ * of the annotation name or a line connecting name to annotatino
```

Comment:
nit (honestly you can ignore anything I post with nit, not worth a rev by itself, just tagging stuff in case it slid by)